packages/frameworks-sveltekit/src/lib/index.ts

```diff
@@ -226,10 +242,21 @@ export async function getSession(
   const data = await response.json()
 
   if (!data || !Object.keys(data).length) return null
-  if (status === 200) return data
+  if (status === 200) return {
+    session: data,
+    cookies: parseCookies(Array.from(response.headers).filter(([headerName]) => headerName === 'set-cookie'))
```

Comment:
can we use the [`getSetCookie`](https://developer.mozilla.org/en-US/docs/Web/API/Headers/getSetCookie) here? 😁

Reply:
Yeah good call, modified that a bit. Also added `cookie and `@types/cookie` as peer deps to `@auth/svelte` since their dependencies in `@auth/core` already anyway and that'll always be installed together. Does that make sense? Or should I just add them as straight-up dependencies for `@auth/sveltekit` too?